<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_manutencao</class>
 <widget class="QWidget" name="create_manutencao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nova Manutecao</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>611</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Id_moto:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_id_moto"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Placa:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_placa"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Tipo de Manutecao:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_tipo_manuntecao">
            <item>
             <property name="text">
              <string>Troca de Oleo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Seguro</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rastreador</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Revisao</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pastilhas de Freio</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pneu</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Outros</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Valor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_valor"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="date_manuntecao"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Km Atual da Moto:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_km_atual"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Observacao</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_observacao"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_salvar">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
